LICENSE.txt

```diff
@@ -19,3 +19,414 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+=======================================================================
+
+Anjin logo's license
+
+(Documentation~/2_Anjin_Logo_white.png#gh-dark-mode-only)
+(Documentation~/1_Anjin_Logo_black.png#gh-light-mode-only)
+
+Attribution-NonCommercial-NoDerivatives 4.0 International
```

Comment:
I think the CC-BY-NC-ND license is too restrictive. Even in commercial contexts, we should allow the use of the Anjin logo in materials like presentation slides or doujin publications that show companies using Anjin. Doing so would help Anjin gain better recognition and promote its growth.

Reply:
I agree, with [c78b153](https://github.com/DeNA/Anjin/pull/158/commits/c78b153899afcb933defbb2ec9b3db1f40278410) , I have changed the license to CC BY-ND.